Index: b/CMakeLists.txt
===================================================================
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -7,7 +7,7 @@ set (CMAKE_CXX_STANDARD 11)
 # Build KStars Lite with -DBUILD_KSTARS_LITE=ON
 option(BUILD_KSTARS_LITE "Build KStars Lite" OFF)
 # Fetch translations with -DFETCH_TRANSLATIONS=ON
-option(FETCH_TRANSLATIONS "Fetch Translations" OFF)
+option(FETCH_TRANSLATIONS "Fetch Translations" On)
 # minimal requirements
 cmake_minimum_required (VERSION 2.8.12 FATAL_ERROR)
 
