Description: Fixes a typo
Author: Herbert Parentes Fortes Neto <hpfn@debian.org>
Last-Update: 2016-11-22
Index: libgphoto2-2.5.11/camlibs/stv0674/stv0674.c
===================================================================
--- libgphoto2-2.5.11.orig/camlibs/stv0674/stv0674.c
+++ libgphoto2-2.5.11/camlibs/stv0674/stv0674.c
@@ -104,7 +104,7 @@ static int file_list_func (CameraFilesys
     result = stv0674_file_count(camera->port, &count);
     if (result < GP_OK)
     {
-	GP_DEBUG("file count returnd %d\n",result);
+	GP_DEBUG("file count returned %d\n",result);
 	return result;
     }
 
