Description: Fixes typo
Author: Herbert Parentes Fortes Neto
Last-Update: 2016-03-26
Index: libgphoto2-2.5.10/camlibs/canon/canon.c
===================================================================
--- libgphoto2-2.5.10.orig/camlibs/canon/canon.c
+++ libgphoto2-2.5.10/camlibs/canon/canon.c
@@ -405,7 +405,7 @@ replace_filename_extension(const char *f
                 return buf;
         } else {
                 GP_DEBUG ("replace_filename_extension: "
-                          "New name for filename '%s' doesnt fit in %s line %i.",
+                          "New name for filename '%s' doesn't fit in %s line %i.",
                           filename, __FILE__, __LINE__);
                 return NULL;
         }
@@ -457,7 +457,7 @@ filename_to_audio(const char *filename,
                 return buf;
         } else {
                 GP_DEBUG ("filename_to_audio: "
-                          "New name for filename '%s' doesnt fit in %s line %i.",
+                          "New name for filename '%s' doesn't fit in %s line %i.",
                           filename, __FILE__, __LINE__);
                 return NULL;
         }
